File: js/src/test/Exchange/test.fetchAccounts.js
import testAccount from './base/test.account.js';
import testSharedMethods from './base/test.sharedMethods.js';
async function testFetchAccounts(exchange, skippedProperties) {
    const method = 'fetchAccounts';
    const accounts = await exchange.fetchAccounts();
    testSharedMethods.assertNonEmtpyArray(exchange, skippedProperties, method, accounts);
    for (let i = 0; i < accounts.length; i++) {
        testAccount(exchange, skippedProperties, method, accounts[i]);
    }
}
export default testFetchAccounts;

File: julia/Ccxt/test/exchange/test.fetchAccounts.jl
using Test
using Ccxt
function testFetchAccounts(exchange, skippedProperties)

    method = "fetchAccounts";
    accounts = Base.fetch(fetchAccounts(exchange));
    assertNonEmtpyArray(testSharedMethods, exchange, skippedProperties, method, accounts);
    i = 0
    while functions.ccxtruthy(functions.ccxt_lt(i, length(accounts)))
        testAccount(exchange, skippedProperties, method, get(accounts, i + 1, nothing));
        i += 1
    end
    return true
end
